database/resources/events.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Hackathon",
+    "description": "What do I need to know for a hackathon?",
```

[Nitpick] I think it would be best to describe what the source is talking about.
```suggestion
    "description": "What do I need to know for a hackathon?",
```